misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import pytz
+
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils.six import StringIO
+from django.utils import timezone
+
+from misago.conf import settings
+from misago.users.management.commands import anonymizeoldips
+from misago.core.utils import ANONYMOUS_IP
+
+
+# - command is not anonymizing user's IP if anonymization is disabled
+# - command is anonymizing user's IP if its old
+
+UserModel = get_user_model()
+
+
+class AnonymizeOldIpsTests(TestCase):
+    def setUp(self):
+        self.user = UserModel.objects.create_user('Bob', 'bob@bob.com', 'pass123')
+        self.user_with_joined_on = UserModel.objects.create_user('Bob1', 'bob1@bob.com', 'pass1231')
+        self.user_with_joined_on.joined_on = datetime.datetime(2013, 11, 20, 20, 8, 7, 127325, tzinfo=pytz.UTC)
```

Comment:
This is incorrect Django code. Django way to get datetime that was, for example, 50 days ago is following:

```python
from datetime import timedelta

from django.utils import timezone


current_datetime = timezone.now()
datetime_50_days_past = current_datetime - timedelta(days=50)
```